content/CORE/CORETutorials/SystemConfiguration/CreatingCAsandCertificates/CreatingCertificateAuthorities.md

```diff
@@ -11,129 +11,109 @@ tags:
+TrueNAS can act as a certificate authority (CA). 
+When encrypting SSL or TLS connections to the TrueNAS system, you can import an existing CA or create a CA and certificate on the TrueNAS system. 
+The certificate appears on the dropdown menus for services that support SSL or TLS. 
 
-Go to **System > CAs** and click **ADD**. Name the CA, then choose the **Type**. The three type options are *Internal CA*, *Intermediate CA*, and *Import CA*. The process for each type is slightly different.
+Go to **System > CAs** and click **ADD**. Enter a name for the CA, then choose the type from the **Type** dropdown list of three, **Internal CA**, **Intermediate CA**, or **Import CA**. The process to add a CA for each type is slightly different.
 
-## Internal CA
+## Creating CA
 
-### Identifier and Type
+A CA must exist in CORE to add an Intermediate CA. This can be an internal or imported CA.
 
-Set *Internal CA* as the **Type**. 
+To create a CA:
 
-You can select a profile for the CA to auto-fill options like **Key Type**, **Key Length**, **Digest Algorithm**. Otherwise, you must set options manually.
+1. Enter or select the **Identifier and Type** setting options.
+   
+   ![AddInternalCAIdentTypeNoProfile](/images/CORE/13.0/AddInternalCAIdentTypeNoProfile.png "Add Internal CA Name and Type")
 
-![Internal CA Identifier Type](/images/CORE/12.0/InternalCAIdentifierType.png)
+   a. Enter a name for this CA.
+   b. Select **Internal CA** from the **Type** dropdown list to create an internal certificate. 
+      Select **Intermediate CA** to create an intermediate certificate. This displays the **Signing Certificate Authority** field in **Certificate Options**.
 
-### Certificate Options
+2. Select an option from the **Profiles** dropdown list. 
+   A profile for the CA auto-fills options like **Key Type**, **Key Length**, **Digest Algorithm**. Otherwise, you must set options manually.
 
-1. Select a **Key Type** from the drop-down. We recommend the *RSA* key type. 
-2. Select the **Key Length**. We recommend a minimum of *2048* for security reasons. 
-3. Select a **Digest Algorithm**. We recommend *SHA256*. 
-4. Enter the **Lifetime** of the CA in days to set how long the CA will remain valid.
+   To add an OpenVPN Root CA, select **OpenVPN Root CA**. 
+   The configuration form populates with default settings, enables **Basic Constraints**, **Authority Key Identifier**, **Extended Key Usage**, and **Key Usage**, and set the options for each extension.
 
-![Internal CA Certificate Options](/images/CORE/12.0/InternalCACertificateOptions.png)
+   ![AddInternalCAOpenVPNRootProfile](/images/CORE/13.0/AddInternalCAOpenVPNRootProfile.png "Add OpenVPN Root CA Profile")
+   
+   To add CA certificate select **CA**. 
+   The configuration form populates with default settings, enables **Basic Constraints**, **Authority Key Identifier**, **Extended Key Usage**, and **Key Usage**, and set the options for each extension.
```

Comment:
```suggestion
   To add a CA certificate, select **CA**. 
   The configuration form populates with default settings, enables **Basic Constraints**, **Authority Key Identifier**, **Extended Key Usage**, and **Key Usage**, and sets each extension's options.
```

Reply:
I'll adjust this, we don't use possessive case 's in articles.